server/controllers.py

```diff
@@ -60,4 +59,18 @@ def get_emotion_by_id(emotion_id : int):
     return {"emotion": emotions[emotion_id]} 
 
 if __name__ == "__main__":
+    
+    model = CNN()
```

Comment:
Initialize the model outside of the main function, to make it accessible for APIs, or make it a global variable.

Reply:
Moved to global scope, thanks.